lib/ae_mdw/aexn_contracts.ex

```diff
@@ -110,6 +111,12 @@ defmodule AeMdw.AexnContracts do
     end
   end
 
+  @spec event_name(AeMdw.Contracts.event_hash()) :: String.t() | nil
```

Comment:
Since we're using it in several places, can we use a new type for these names? `event_name()` maybe

Reply:
Only two places but I know you like named type specs so here it is (: